tutorials/cloud-iot-rtdp/index.md

```diff
@@ -180,10 +187,23 @@ Core to receive data from MQTT clients.
     Warning: This tutorial includes a public/private key pair for testing
     purposes. Do not use this pair in a production environment.
 
+## Create threshold values in DataStore
+In this section, you insert threshold values for each of the devices, registered in the IoT Core Device Manager, in the DataStore.
+1. In Cloud Shell, run python script to insert the device objects into Datastore:
+
+        sudo pip install google-cloud-datastore
```

Comment:
I prefer to use virtualenv to running pip as root, e.g.:

    virtualenv env && source env/bin/activate
    pip install -r requirements.txt 
    ...